pkg/security/secl/compiler/ast/secl.go

```diff
@@ -39,12 +40,9 @@ any = "\u0000"…"\uffff" .
 )
 
 func unquotePattern(t lexer.Token) (lexer.Token, error) {
-	unquoted, err := strconv.Unquote(t.Value[1:])
-	if err != nil {
-		return t, participle.Errorf(t.Pos, "invalid pattern string %q: %s", t.Value, err)
-	}
+	unquoted := strings.TrimSpace(t.Value[1:])
```

Comment:
Why do we use `t.Value[1:]` and not simply `t.Value` here ?

Reply:
in order to remove the `r` of the expression r".*" for example.

Reply:
Oh yeah of course